test/pls.jl

```diff
@@ -217,6 +229,10 @@ end
     @test fm1.optsum.initial == ones(3)
     @test lowerbd(fm1) == zeros(3)
 
+    spL = sparseL(fm1)
+    @test size(spL) == (4114, 4114)
+    @test 733090 < nnz(spL) < 733100
```

Comment:
are the bounds here just because of floating point and whether some values are rounded off to zero?

Reply:
Yes.  I forget the details but I did encounter changes.  At one time I had a call to `dropzeros!` in the `sparseL` method but it is not certain it is a good idea to drop isolated zeros in the middle of a dense block so I allowed for that the be changed without affecting the test.